examples/example-gateway/idl/clients/bar/bar.thrift

```diff
@@ -179,6 +193,18 @@ service Bar {
         zanzibar.http.status = "200"
     )
 
+    BarResponse argWithQueryParams(
+        1: required string name	(zanzibar.http.ref = "query.name")
+        2: optional string userUUID	(zanzibar.http.ref = "query.userUUID")
+        3: optional list<string> foo	(zanzibar.http.ref = "query.foo")
+        4: required list<i8> bar	(zanzibar.http.ref = "query.bar")
+        // TODO(argouber): Actually allow sets - preferably with go.tag of slice
```

Comment:
Removed the optional field that was not being set in the test.  We need a good way to specify set in the query parameter.  Will do this in a separate PR